### PR TITLE
Fixes misc

### PR DESCRIPTION
**Pull request by wwysoc2**
Only the changed pages have been reviewed.  Will create a PR later for the others.

- Changed some wording
- Fixed places where docs didn't render correctly
- Tried to fix quick_start -- getting_started mixup (let me know if I miss any)

Planned PR: Change order of menu list.

### CODE REVIEW

**Review by clsackett — COMMENTED** [review]
Minor grammatical fixes, otherwise LGTM!

File: docs/Data_Portal/Users_Guide/Projects.md [hunk 1]
```diff
@@ -42,15 +42,15 @@ Clicking the link for each project name on the table will bring users to that sp
...
 [![Project Entity](images/ProjectEntity.png)](images/ProjectEntity.png "Click to see the full image.")
 
-Four buttons on the left of the header allow the user to perform a variety of actions related to the project: 
+Four buttons on the left of the header allow the user to perform a variety of actions related to the project:
 
 * __Save New Cohort__: Creates a new cohort consisting of all the cases in the project
 * __Biospecimen__: Downloads biospecimen metadata associated with all cases in the project in either TSV or JSON format
 * __Clinical__: Downloads clinical metadata about all cases in the project in either TSV or JSON format
-* __Manifest__: Downloads a manifest for all data files available in the project. The manifest can be used with the GDC Data Transfer Tool to download the files.
+* __Manifest__: Downloads a manifest for all data files available in the project. The manifest can be used with the GDC Data Transfer Tool to download the files
```
Comment (wteouchicago) [inline docs/Data_Portal/Users_Guide/Projects.md:50]: maybe I got the rule wrong for when to put periods. I thought if the bullet contains >1 sentences, we add periods to all of them? Or do we just add to all except the last sentence?

File: docs/Data_Portal/Users_Guide/Repository.md [hunk 2]
```diff
@@ -2,38 +2,37 @@
...
 [![Main Cohort Toolbar](images/MainCohortToolbar.png)](images/MainCohortToolbar.png "Click to see the full image.")
 
-The active cohort can be changed by clicking the cohort name and choosing a new one from the dropdown menu.  If the cohort of interest does not appear in the list, try [creating a new cohort.](quick_start.md#main-toolbar)
-
 For users who want to browse all files that are available at the GDC, create a new cohort via the main toolbar and use it with the Repository tool.
 
 ## Filtering a Set of Files ##
 
-Users may only be interested in browsing through or downloading a subset of files associated with the current cohort. For this purpose, a set of commonly-used default facet cards is provided in the left panel of the Repository tool to allow users to filter the files presented in the table on the right. The facet cards are as follow:
+As most users are searching for specific type of file, a set of commonly-used default facet cards can be used in the left panel of the Repository tool to allow users to filter the files presented in the table on the right. The facet cards are as follow:
```
[the remark this answers is not in the document]
Comment (wwysoc2) [reply docs/Data_Portal/Users_Guide/Repository.md]: Fixed. The "s" button must have been broken on my keyboard!

File: docs/Data_Portal/Users_Guide/cohort_builder.md [hunk 3]
```diff
@@ -42,11 +41,11 @@ The first six (or fewer) filters are shown for each card, but can be expanded to
 
 ### Available Data Filters
 
-Toward the bottom of the list of filter categories, "Available Data" can be selected.  These filters differ from the other default filters as it allows for only cases that have certain types of associated data files to be displayed. For example, filtering in the "Experimental Strategy" card for "RNA-Seq" will only display cases in the active cohort that have associated files with an experimental strategy of "RNA-Seq". These filters may be useful for analyses that requires only cases for which a certain type of analysis was performed.
+Toward the bottom of the list of filter categories, "Available Data" can be selected.  These filters differ from the other default filters as they allow for cases that have certain types of associated data files. For example, filtering in the "Experimental Strategy" card for "RNA-Seq" will only display cases in the active cohort that have associated RNA-Seq files. These filters may be useful for analyses that requires only cases for which a certain type of analysis was performed.
```
[the remark this answers is not in the document]
Comment (wwysoc2) [reply docs/Data_Portal/Users_Guide/cohort_builder.md]: Fixed.  Changed sentence wording a bit.

File: docs/Data_Portal/Users_Guide/Projects.md (hunk 1 above)
Comment (wwysoc2) [reply docs/Data_Portal/Users_Guide/Projects.md:50]: I think I just robotically eliminated them.  The > 1 sentence rule sounds reasonable.

**Comment by clsackett** [comment]
LGTM!

File: docs/Data_Portal/Users_Guide/Repository.md [hunk 4]
```diff
@@ -2,38 +2,37 @@
...
 ## Filtering a Set of Files ##
 
-Users may only be interested in browsing through or downloading a subset of files associated with the current cohort. For this purpose, a set of commonly-used default facet cards is provided in the left panel of the Repository tool to allow users to filter the files presented in the table on the right. The facet cards are as follow:
+As most users are searching for specific types of files, a set of commonly-used default facet cards can be used in the left panel of the Repository tool to allow users to filter the files presented in the table on the right. The facet cards are as follows:
 
 * **Experimental Strategy**: Experimental strategies used for molecular characterization of the cancer
 * **WGS Coverage**: Range of coverage for WGS aligned reads
 * **Data Category**: A high-level data file category, such as "Raw Sequencing Data" or "Transcriptome Profiling"
-* **Data Type**: Data file type, such as "Aligned Reads" or "Gene Expression Quantification". Data Type is more granular than Data Category.
+* **Data Type**: Data file type, such as "Aligned Reads" or "Gene Expression Quantification". Data Type is more granular than Data Category
```
Comment (wteouchicago) [inline docs/Data_Portal/Users_Guide/Repository.md:29]: nitpick - add period at end?

**Review by wteouchicago — APPROVED** [review]

**Pull request merged**